README.md

```diff
@@ -214,9 +234,11 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 | <a href="https://github.com/MariaLetta/mega-doodles-pack/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184917535-88553471-26eb-4bf2-a532-b719341a66c8.png' alt="Mega_Doodles_Pack_free" /></a> | [Mega Doodles Pack](https://github.com/MariaLetta/mega-doodles-pack/)  <br><br> ✔ Free <br><br> |
+| <a href="https://www.drawkit.com/product/mental-health-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194963389-3399dedb-e379-462e-890b-bbe449b8d9da.png' alt="Mental_Health_icons_free_illustrations" /></a> | [Mental Health Icons](https://www.drawkit.com/product/mental-health-icons)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit                                                 |
 | <a href="https://drawkit.com/product/mental-health-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193481139-ef3f5fca-4514-4fed-8b88-8ad6ab27580b.png' alt="Mental_Health_free_illustrations" /></a> | [Mental Health](https://drawkit.com/product/mental-health-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br><br> ✌️Author <br> DrawKit                                                 |
 | <a href="https://www.figma.com/community/file/1000311109311441524/Nikuu-3d-Illustration-Pack-by-Paperpillar"><img width='700' src='https://user-images.githubusercontent.com/74038190/184643525-abc73b4f-9458-4f50-8c1a-ddb8b6a2c2a1.png' alt="Nikuu_3D_free_illustrations" /></a> | [Nikuu 3D](https://www.figma.com/community/file/1000311109311441524/Nikuu-3d-Illustration-Pack-by-Paperpillar)  <br><br> ✔ Free <br><br>Format <br> • Figma<br><br> ✌️Author <br> Paperpillar                                                                                                       |
 | <a href="https://maryamato88.gumroad.com/l/pbmkt"><img width='700' src='https://user-images.githubusercontent.com/74038190/184478348-653d118b-8797-4566-9a68-cb1fb981b1eb.png' alt="Notion_Style_free_illustrations" /></a> | [Notion-Style](https://maryamato88.gumroad.com/l/pbmkt)  <br><br> ✔ Free <br><br>Format <br> • SVG<br>• PNG<br><br> ✌️Author <br>Mary Amato                                                                      |
+| <a href="https://www.drawkit.com/product/nutrition-icons"><img width='700' src='https://user-images.githubusercontent.com/34191321/194963072-2a0c777d-e24e-48d7-8156-b185888eb03a.png' alt="nutrition_icons_free_illustrations" /></a> | [Nutrition Icons](https://www.drawkit.com/product/nutrition-icons)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit |
```

Comment:
- change the src link to `https://user-images.githubusercontent.com/74038190/195570876-ffac6d16-8f67-4899-bed2-84cefe6b1378.png`